{"name": "test_divide_number", "status": "skipped", "statusDetails": {"message": "XFAIL reason: Need To Investigate\n\n_pytest.outcomes.XFailed: Need To Investigate", "trace": "self = <DemoPytest.test_fail.Test_Math object at 0x00000228B83DBE30>\n\n    def test_divide_number(self):\n>     pytest.xfail(\"Need To Investigate\")\nE     _pytest.outcomes.XFailed: Need To Investigate\n\nDemoPytest\\test_fail.py:5: XFailed"}, "start": 1710442254970, "stop": 1710442254970, "uuid": "bbdf476c-f21e-4c83-9bc9-518efc4fc5a2", "historyId": "e0d269bca92a8722eaf6a7a0eec1f233", "testCaseId": "e0d269bca92a8722eaf6a7a0eec1f233", "fullName": "DemoPytest.test_fail.Test_Math#test_divide_number", "labels": [{"name": "parentSuite", "value": "DemoPytest"}, {"name": "suite", "value": "test_fail"}, {"name": "subSuite", "value": "Test_Math"}, {"name": "host", "value": "LAPTOP-U9R56RQR"}, {"name": "thread", "value": "15700-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "DemoPytest.test_fail"}]}